lib/features/family/features/reflect/presentation/pages/summary_screen.dart

```diff
@@ -198,12 +200,25 @@ class _SummaryScreenState extends State<SummaryScreen> {
 
   FunButton getFunButton(SummaryDetails details) {
     const text = 'Done';
-    void onTap() {
+
+    Future<void> onTap() async {
+      // If second game
+      var gameplays = await _cubit.getTotalGamePlays();
```

Comment:
The widget is doing too much logic now. The cubit should be the one comparing the total gameplays with the store review game count.

Reply:
I'll move that! :)